File: Clobo.Domain/Entities/Ticket.cs
using System;
using Clobo.Domain.Base;

namespace Clobo.Domain.Entities
{
    public class Ticket : TrackableEntity
    {
        public Ticket()
        {
        }

        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public Agent? Agent { get; set; }
        public Team? Team { get; set; }
        public TicketStatus TicketStatus { get; set; }
        public Customer Customer { get; set; }
        public Product Product { get; set; }
        public IList<TicketNote> TicketNotes { get; set; }
    }
}

File: clobo.client/clobo/src/models/entities/Ticket.ts
import { Agent } from "./Agent";
import { Customer } from "./Customer";
import { Product } from "./Product";
import { Team } from "./Team";
import { TicketNote } from "./TicketNote";
import { TicketStatus } from "./TicketStatus";

export interface Ticket{
    id:number;
    name:string;
    description:string;
    agent?:Agent;
    team?:Team;
    ticketStatus: TicketStatus;
    customer:Customer;
    product:Product;
    ticketNotes: TicketNote[];
}
